packages/l10n_languages/CHANGELOG.md

```
@@ -1,3 +1,12 @@
+## 1.0.0
```

Markdownlint (0.37.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)